Collections/Game.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+enum Enum {
+    GENRE1, GENRE2, GENRE3
+}
+
+public class Game {
+    private String title;
+    private Enum genre;
+    private Double price;
+    private Integer copiesSold;
+
+    public Double getPrice() {
+        return price;
+    }
+
+    public Game(String title, Enum genre, Double price, Integer copiesSold) {
+        this.title = title;
+        this.genre = genre;
+        this.price = price;
+        this.copiesSold = copiesSold;
+    }
+
+    @Override
+    public String toString() {
+        return "Game{" +
+                "title='" + title + '\'' +
+                ", genre=" + genre +
+                ", price=" + price +
+                ", copiesSold=" + copiesSold +
+                '}';
+    }
+
+    public static void main(String[] args) {

```

Comment:
Do not add everything into one class. Create separate class for enum, for the Game and for the application itself to run from.

Reply:
ok